### DISCUSSION TITLE
Question: Any guidance or documentation

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/winget-cli#619] [Q&A] [answered]
New to this. Is there any particular documentation or guidance on how or whether this can be used for LOB apps that are distributed from the Store for specific tenants?

**Comment by denelon** [comment, marked as answer]
We don't have support for all of the store capabilities. The current experimental store feature is limited to a set of about 300 developer apps that are free and rated E for everyone. The REST API is being developed so that an enterprise can stand up a private repository.

**Reply by myokeeh** [reply]
With the release of Windows 11 on Oct 5 and this post (https://techcommunity.microsoft.com/t5/windows-it-pro-blog/evolving-the-microsoft-store-for-business-and-education/ba-p/2569423), it appears winget is what they're pushing as a stop-gap so I can get Windows 11 clients to obtain apps, but after testing it, I believe the author of that post doesn't know this doesn't work. Is there something new to allow this?

**Reply by denelon** [reply]
The work hasn't all been completed. The section on "How we're getting there" covers future Intune Integration. There are subtle differences between a push model and a pull model for package management. 

The new experimental source for the store in yesterday's [Pre-release](https://github.com/microsoft/winget-cli/releases/tag/v1.0.12576) includes access to most of the Apps in the store.

**Reply by myokeeh** [reply]
@denelon Has this improved since September?

**Reply by denelon** [reply]
The Windows Package Manager now ships with the Microsoft Store "msstore" source. Most of the packages in the store are available for developers to install (free and rated "e" for everyone). The work to integrate with Intune is in progress. There are still plenty of [Issues](https://github.com/microsoft/winget-cli/issues?q=is%3Aissue+is%3Aopen+label%3Amsstore) to be resolved.
